integration_tests/features/flow_run.feature

```diff
@@ -138,3 +138,30 @@ Feature: `flow run` command
       | zendesk_ticket_data |
     And the script zendesk_ticket_data.check_extra.py output file has the lines:
       | extra_col: yes |
+
+  Scenario: fal flow run command with exclude arg
```

Comment:
can you add a test that has both `--select` and `--exclude` at the same time

Reply:
Sure